utils/constants/vm_ids.go

```diff
@@ -5,8 +5,37 @@ package constants
 
 import "github.com/ava-labs/avalanchego/ids"
 
+const (
+	PlatformVMName = "platformvm"
+	AVMName        = "avm"
+	EVMName        = "evm"
+	SubnetEVMName  = "subnetevm"
+	XSVMName       = "xsvm"
```

Comment:
not sure I like having non-Primary-Network VMs here, but I guess there is no alternative?